src/core_plugins/kibana/public/context/api/utils/sorting.js

```diff
@@ -1,5 +1,33 @@
 import _ from 'lodash';
 
+
+/**
+ * The list of field names that are allowed for sorting, but not included in
+ * index pattern fields.
+ *
+ * @constant
+ * @type {string[]}
+ */
+const META_FIELD_NAMES = ['_seq_no', '_doc', '_uid'];
+
+/**
+ * Returns a field from the intersection of the set of sortable fields in the
+ * given index pattern and a given set of candidate field names.
+ *
+ * @param {IndexPattern} indexPattern - The index pattern to search for
+ *     sortable fields
+ * @param {string[]} fields - The list of candidate field names
+ *
+ * @returns {string[]}
+ */
+function getFirstSortableField(indexPattern, fieldNames) {
+  const sortableFields = fieldNames.filter((fieldName) => (
+    META_FIELD_NAMES.includes(fieldName)
+    || (indexPattern.fields.byName[fieldName] || { sortable: false }).sortable
```

Comment:
The fatal error being shown is a consequence of #12124, which I'm trying to fix as well right now. This has proven to be more difficult than expected, but as soon as I have worked around the error handling memory leaks in courier I will handle these more gracefully.